xml/System.IO.Compression/ZipFile.xml

```diff
@@ -578,12 +629,12 @@
       </Docs>
     </Member>
     <Member MemberName="ExtractToDirectory">
-      <MemberSignature Language="C#" Value="public static void ExtractToDirectory (string sourceArchiveFileName, string destinationDirectoryName, System.Text.Encoding entryNameEncoding, bool overwriteFiles);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void ExtractToDirectory(string sourceArchiveFileName, string destinationDirectoryName, class System.Text.Encoding entryNameEncoding, bool overwriteFiles) cil managed" />
+      <MemberSignature Language="C#" Value="public static void ExtractToDirectory (string sourceArchiveFileName, string destinationDirectoryName, System.Text.Encoding entryNameEncoding, bool overwrite);" />
```

Comment:
if you we're going to change the source to match the ref, we shouldn't change the names here right?

Reply:
Uh oh... Did I do it backwards?

Reply:
Gah, my bad, I did it backwards. Let me fix it and will submit a new iteration.
corefx\src\System.IO.Compression.ZipFile\ref\System.IO.Compression.ZipFile.cs

Reply:
Ok fixed.